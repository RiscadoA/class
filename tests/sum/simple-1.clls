proc main() {
    cut {
        println("case x");
        case x of {
            |#C: println("close x"); close x
            |#W: println("wait x 1"); wait x; ()
        }
        |x : choice of { |#C: wait |#W: close }|
        println("#C x");
        #C x;
        println("wait x 2");
        wait x;
        ()
    }
};;
