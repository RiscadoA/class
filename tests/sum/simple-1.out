#C x
case x
close x
wait x 2
